doc/database-model.md

```diff
@@ -0,0 +1,61 @@
+
+# Database Entity Relationship Diagram
+
+```mermaid
+erDiagram
+    USER {
+        String _id PK
+        String name
+        String googleID
+        String address
+        ObjectId[] dogs
+    }
+    DOG {
+        String _id PK
+        String name
+        String breed
+        Number age
+        Date birth
+        Number weight
+        Boolean neutered
+        String medication
+        Number medicationPurchaseFrequency
+        String vetCardImageUrl
+        String qrChipImageUrl
+        String[] allergies
+        String[] illness
+        ObjectId[] foods
+        ObjectId[] medical_history
+        ObjectId[] appointments
+        ObjectId[] photos
+    }
+    
+    MEDICAL_HISTORY {
+        String _id PK
+        Date date
+        String vaccine
+        String vet
+    }
+    APPOINTMENT {
+        String _id PK
+        Date date
+        String vet
+        String reason
+    }
+    PHOTO {
```

Comment:
Why this entity?

Reply:
if we need to upload a image for pets it's a better solution add a field image in pets which would be an array of strings or objects

Reply:
perfect. thanks